pkg/types/installconfig.go

```diff
@@ -86,6 +86,7 @@ const (
 )
 
 //go:generate go run ../../vendor/sigs.k8s.io/controller-tools/cmd/controller-gen crd:crdVersions=v1 paths=. output:dir=../../data/data/
+//go:generate go run ../../vendor/k8s.io/code-generator/cmd/deepcopy-gen --output-file zz_generated.deepcopy.go ./...
```

Comment:
Here is where the generator is actually invoked.  This ties in with existing codegen, and `hack/verify_codegen.sh`